webapp/exercise_admin/templates/exercise_admin/base.html

```diff
@@ -178,36 +186,35 @@ <h2 id="edit-feedback-div-title">Edit feedback question:</h2>
         <div class="generalinfo-right-side">
           <h2>Feedback questions</h2>
           <div class="bordered-editable-area">
-            <div id="feedback-question-table" class="feedback-table-div">
-              <table class="feedback-question-table">
+            <div id="feedback-question-table-div" class="feedback-table-div">
+              <table id="feedback-question-table" class="feedback-question-table">
```

Comment:
Corresponding change required to submit_main_form function (section commented as "Get the linked feedback questions").